values.yaml

```diff
@@ -111,7 +111,7 @@ security:
 
 image:
   repository: docker.io/datawire/aes
-  tag: 1.5.5
+  tag: 1.6.0
```

Comment:
@alexgervais `docker.io/datawire/aes:1.6.0` does not exist... yet?

Reply:
@inercia Indeed, it does not exist yet. This PR is in preparation for the 1.6.0 release, following our typical release process.

Reply:
Ok, I have updated it to a _draft_. Feel free to revert the status when 1.6.0 is released.